test/io/CsvWriterTest.ooc

```diff
@@ -30,7 +33,21 @@ CsvWriterTest: class extends Fixture {
 			}
 			reader free()
 			outputFilename free()
+			// This needs to be moved if any more tests are implemented
+			this _removeOutputDirectory()
```

Comment:
In `.gitignore`:
```
# Test output
/test/draw/output
```
Just add  `/test/io/output`, or maybe use a wildcard: `/test/*/output` to ignore all output files in all tests.